src/Illuminate/Foundation/Testing/TestResponse.php

```diff
@@ -172,6 +172,21 @@ public function assertRedirect($uri = null)
         return $this;
     }
 
+    /**
+     * Assert that the response is empty.
+     *
+     * @return $this
+     */
+    public function assertEmpty()
+    {
+        PHPUnit::assertTrue(
+            $this->isEmpty(),
+            'Response status code ['.$this->getStatusCode().'] is not a empty response status code.'
```

Comment:
This message doesn't make any sense?

Reply:
For assertion method called `assertEmpty` it only makes sense for the error message to be `"Response is not empty"`.

But since here he checks status codes maybe something like `"Status code [{$this->getStatusCode()}] does not account for no content response"` would be more telling?

Reply:
When I wrote this message I tried to follow the same idea of ​​the messages of the methods `assertForbidden` or `assertUnauthorized`, following the idea, I think the messages needs to be something like: `'Response status code ['.$actual.'] is not an empty status code.`, what do you think?

Reply:
It is not correct because it should say the message is non-empty. The status code has nothing to do with it at all.

Reply:
@GrahamCampbell Of course the method has relation with status code, if you decide to look what the `'isEmpty` method does you'll see:

```php
public function isEmpty(): bool
    {
        return \in_array($this->statusCode, [204, 304]);
    }
```

Reply:
Oh. I assumed it would be testing if the body was empty. What a strange name for the method then!